jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -477,6 +478,23 @@ public void setShowMessageInTitle(boolean showMessageInTitle)
         _showMessageInTitle = showMessageInTitle;
     }
 
+    /**
+     * @return The mime type to be used when a client does not specify an Accept header, or the request did not fully parse
+     */
+    @ManagedAttribute("Mime type to be used when a client does not specify an Accept header, or the request did not fully parse")
+    public Type getDefaultResponseMimeType()
+    {
+        return _defaultResponseMimeType;
+    }
+
+    /**
+     * @param defaultResponseMimeType The mime type to be used when a client does not specify an Accept header, or the request did not fully parse
+     */
+    public void setDefaultResponseMimeType(Type defaultResponseMimeType)
+    {
+        _defaultResponseMimeType = defaultResponseMimeType;
```

Comment:
```suggestion
        _defaultResponseMimeType = Objects.requireNonNull(defaultResponseMimeType);
```

Reply:
Type is changed to String and also uses the `requireNonNull` helper now